### ISSUE TITLE
`VGSTextField` and `VGSLabel` have no intrinsicContentSize, requires manually setting height & width

### ISSUE THREAD

**Issue by joshuakcockrell** [verygoodsecurity/vgs-collect-ios#363]
**`VGSTextField` and `VGSLabel` have no intrinsicContentSize, requires manually setting height & width**

**To Reproduce**

```swift   
let label = VGSTextField()

let cardConfiguration = VGSConfiguration(
    collector: CollectCard.shared,
    fieldName: "data.attributes.cvv2"
)
cardConfiguration.type = .cvc
cardConfiguration.isRequiredValidOnly = true

label.font = UIFont(name: "SourceCodePro-Regular", size: 16)
label.placeholder = "TEST"

print(label.intrinsicContentSize) // (-1.0, -1.0) <-- The size is totally broken!
print(label.subviews[0].intrinsicContentSize) // (38.6, 22.0) <-- But subviews have a size for some reason?
```

**Expected behavior**
The VGSTextField should have `intrinsicContentSize`

**Environment (please complete the following information):**
 - SDK Version `1.15`
 - iOS Version: `iOS 16 & 17`
 - Xcode Version: `Xcode 14 & 15`
 - Device: iPhone XS Max
 - Dependency manager and it's version:
    - [ ] Swift Package Manager

**Additional context**
This makes it really hard to use the UI elements with SwiftUI. You basically need to create a duplicate hidden `Text` element and overlay the VGSCollect elements on top:

```swift
Text("000")
    .font(.EMMonoRegular16)
    .lineLimit(0)
    .dynamicTypeSize(.large) // VGSLabel doesn't respect accessibility sizes, fix to large
    .fixedSize()
    .hidden()
    .overlay {
        VGSCollectCVCRepresentable()
    }
```

Why is this the case? Is this some intentional decision to avoid monitoring the size to avoid giving away info about what's contained within?

Even if this was the case, couldn't the VGS UI elements just take up a fixed size? If it's a cvc, then just force the TextField or Label to be 3 characters wide? That's essentially what I'm doing with the `.overlay` hack.

Let's take this to the extreme: someone uses a custom font where each character is a different bit significance and sums the entire width of the `VGSLabel` to determine the exact text contents. So we'd want to obscure the size.. Couldn't VGS provide some `forceMonospace` option that disables this in exchange for surfacing content size? Or maybe the size is only revealed if you use a built-in monospace font that VGS controls? Or perhaps just provide some `sizedToFixText = "000"` argument that would match the size of whatever you pass in?

Most of my time with VGS is spent dealing with these layout issues (as opposed to the actual sending/receiving of data), so this makes the library quite frustrating to work with. [Even the sample apps just resort to manually setting height](https://github.com/vgs-samples/vgs-collect-show-ios-swiftUI-demo/blob/a0f36eb6fe79d9cb6abe2afd09efc4cb549b896d/VGSCollectShowSwiftUIDemo/VGSCollectView/VGSCollectView.swift#L57-L58)

Thanks for any guidance here!

**Comment by joshuakcockrell** [comment]
@dmytrokhl I just tried out the brach. That's working perfect! Thank you. No more need to manually size the label.

Would it be easy to do the same update for `VGSLabel` in VGSShow?

**Comment by dmytrokhl** [comment]
@joshuakcockrell sure we will update VGSLabel too. I will let you know when we release the update.

By the way we working on adding SwiftUI views, but I think it will be released no earlier than end of the year.

**Comment by dmytrokhl** [comment]
@joshuakcockrell `intrinsicContentSize` attribute is available now in VGSCollectSDK [v1.5.1](https://github.com/verygoodsecurity/vgs-collect-ios/releases/tag/1.15.1) and VGSShowSDK [v1.1.7](https://github.com/verygoodsecurity/vgs-show-ios/releases/tag/1.1.7).

**Comment by joshuakcockrell** [comment]
@dmytrokhl I've just tested on VGSCollectSDK and VGSShowSDK and this is working great. Much easier to use with SwiftUI now. Thank you!
